src/modules/admin/admin-tab-categories/AdminTabCategories.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useCallback } from 'react';
+import { NavLink, useRouteMatch, useHistory } from 'react-router-dom';
+
+import { Button as MuiButton } from '@material-ui/core';
+import UnfoldMoreIcon from '@material-ui/icons/UnfoldMore';
+
+import { Button } from 'ui';
+
+import { Url } from 'utils';
+
+import AdminTabCategory from '../models';
+
+import csx from './AdminTabCategories.scss';
+
+const ADMIN_TAB_CATEGORIES: AdminTabCategory[] = [
```

Comment:
You should create utils file and export this in root module index.tsx file - and this ADMIN_TAB_CATEGORIES- should be renamed to CATEGORIES we know this is connected to admin because of location

Reply:
There is already model directory in modules/admin.
Maybe should I locate Categories in Category in Categories.ts?